uclibc add sys pci.h patch

Copyright (C) 2015 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

uClibc is missing the <sys/pci.h> header file.  This file is very
basic and simply includes <linux/pci.h>.

diff --git a/patches/uClibc/0.9.32.1/300-sys-pci.h.patch b/patches/uClibc/0.9.32.1/300-sys-pci.h.patch
new file mode 100644
index 0000000..d6aae31
--- /dev/null
+++ b/patches/uClibc/0.9.32.1/300-sys-pci.h.patch
@@ -0,0 +1,27 @@
+--- uClibc-0.9.33.2/include/sys/pci.h	1969-12-31 16:00:00.000000000 -0800
++++ uClibc-new/include/sys/pci.h	2015-01-07 14:04:28.285091000 -0800
+@@ -0,0 +1,24 @@
++/* Copyright (C) 1997-2014 Free Software Foundation, Inc.
++   This file is part of the GNU C Library.
++
++   The GNU C Library is free software; you can redistribute it and/or
++   modify it under the terms of the GNU Lesser General Public
++   License as published by the Free Software Foundation; either
++   version 2.1 of the License, or (at your option) any later version.
++
++   The GNU C Library is distributed in the hope that it will be useful,
++   but WITHOUT ANY WARRANTY; without even the implied warranty of
++   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
++   Lesser General Public License for more details.
++
++   You should have received a copy of the GNU Lesser General Public
++   License along with the GNU C Library; if not, see
++   <http://www.gnu.org/licenses/>.  */
++
++#ifndef _SYS_PCI_H
++#define _SYS_PCI_H	1
++
++/* We use the constants from the kernel.  */
++#include <linux/pci.h>
++
++#endif /* sys/pci.h */
diff --git a/patches/uClibc/0.9.33.2/300-sys-pci.h.patch b/patches/uClibc/0.9.33.2/300-sys-pci.h.patch
new file mode 100644
index 0000000..d6aae31
--- /dev/null
+++ b/patches/uClibc/0.9.33.2/300-sys-pci.h.patch
@@ -0,0 +1,27 @@
+--- uClibc-0.9.33.2/include/sys/pci.h	1969-12-31 16:00:00.000000000 -0800
++++ uClibc-new/include/sys/pci.h	2015-01-07 14:04:28.285091000 -0800
+@@ -0,0 +1,24 @@
++/* Copyright (C) 1997-2014 Free Software Foundation, Inc.
++   This file is part of the GNU C Library.
++
++   The GNU C Library is free software; you can redistribute it and/or
++   modify it under the terms of the GNU Lesser General Public
++   License as published by the Free Software Foundation; either
++   version 2.1 of the License, or (at your option) any later version.
++
++   The GNU C Library is distributed in the hope that it will be useful,
++   but WITHOUT ANY WARRANTY; without even the implied warranty of
++   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
++   Lesser General Public License for more details.
++
++   You should have received a copy of the GNU Lesser General Public
++   License along with the GNU C Library; if not, see
++   <http://www.gnu.org/licenses/>.  */
++
++#ifndef _SYS_PCI_H
++#define _SYS_PCI_H	1
++
++/* We use the constants from the kernel.  */
++#include <linux/pci.h>
++
++#endif /* sys/pci.h */
